prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -466,21 +466,24 @@ ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_pee
  * If data is copied by read. The progress engine will call the
  * event handler.
  */
-static inline
-ssize_t rxr_pkt_post_read_to_copy_data(struct rxr_ep *rxr_ep,
-				       struct rxr_pkt_entry *pkt_entry,
-				       char *data, size_t data_size,
-				       struct rxr_rx_entry *rx_entry,
-				       size_t data_offset)
+ssize_t rxr_pkt_post_local_read(struct rxr_ep *rxr_ep,
+				struct rxr_local_read_entry *local_read_entry)
 {
 	ssize_t err;
 	int iov_idx;
-	size_t iov_offset, pkt_offset;
-	void *dest_buf;
+	size_t iov_offset, pkt_offset, data_size, data_offset;
```

Comment:
Can you change this function so that `local_read_entry` is used directly? You can shorten the name of this variable if the lines get too long.

Reply:
Done.